<<<<<<< HEAD
package part34_Multiple_Exceptions;

import java.io.File;
import java.io.FileNotFoundException;
import java.io.IOException;
import java.text.ParseException;

public class Test {

	public void run() throws IOException, ParseException {
		// public void run() {

		// throw new IOException();

		// throw new ParseException("Error in command list.", 3);

		new File("test2.txt");

	}

	public void input() throws FileNotFoundException, IOException {

	}

}
=======
package part34_Multiple_Exceptions;

import java.io.File;
import java.io.FileNotFoundException;
import java.io.IOException;
import java.text.ParseException;

public class Test {

	public void run() throws IOException, ParseException {
		// public void run() {

		// throw new IOException();

		// throw new ParseException("Error in command list.", 3);

		new File("test2.txt");

	}

	public void input() throws FileNotFoundException, IOException {

	}

}
>>>>>>> main
